packer/main.pkr.hcl

```diff
@@ -96,6 +164,14 @@ build {
     ]
   }
 
+  source "source.qemu.kvm" {
+    name             = "ubuntu-aarch64"
+    vm_name          = "image.qcow2"
+    iso_url          = "${local.ubuntu_aarch64_info.mirror_url}/${local.ubuntu_aarch64_info.iso_file}"
+    iso_checksum     = "file:${local.ubuntu_aarch64_info.mirror_url}/SHA256SUMS"
+    output_directory = "${local.artifact_dir_prefix}ubunutu-aarch64"
```

Comment:
Probably "ubuntu" here

Reply:
Ugh made this mistake once and then the editor just always autocompleted it after... Guess this one didn't break the build so it didn't get fixed.